### PR TITLE
Decentralization Autonomy Organisation - Keynote  jaya klara brekke - Zettelkasten Notes 

### PR DESCRIPTION
**Pull request by AndreasS2501**
Hi @rufuspollock  I put here the Key note notes.  Lets see if we can find a format how to fit them into your notes/strucutre.

### CODE REVIEW

File: notes/decentralization-autonomy-organisation-brekke.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Overview
```
Comment (rufuspollock) [inline notes/decentralization-autonomy-organisation-brekke.md:1]: I think you can skip the overview/preamble and just have the main heading (or if you want to keep that move to bottom).

File: notes/decentralization-autonomy-organisation-brekke.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+- activsm times: affinity groups groups arround tactics
+- her history background from activism, small decentralized groups, organising, event in spain when... realizing: " in a meeting about how to hold meetings - process fetich - process fatigue (nearly? traumatized her)
+- "i think it's because i simply burnt out around the kind of fascination and the focus on organizing and arranging processes and procedures in the abstract" I talked about this with aeon in a very similar if not same way, she calls it: "process fetich - process fatigue", I called it the "conference hopper" symptoms ( are you really doing something or just talking about doing something)
+- " and i very soon found myself in assemblies about how to run assemblies in a kind of infinite regression"
+- " that there's no neutral means(the medium shapes the esperience - mcluahn)  through which to to solve differences the mediums really matter and in this in this case it was the mediums were assemblies and it was very clear that there were certain people that suited that form of decision-making and that form of resolving differences and conducting politics better than others(she includes herself with the others) "
```
Comment (rufuspollock) [inline notes/decentralization-autonomy-organisation-brekke.md:21]: small formatting here: `means(the medium`

File: notes/democracy.earth.md [hunk 3]
```diff
@@ -4,4 +4,6 @@
 
 Why does it always involve some kind of financialization?
 
-![[Pasted image 20220120225319.png]]
\ No newline at end of file
+![[Pasted image 20220120225319.png]]
+
+I would be very interested in talking with Santi Siri from Democracy earth. Where does he see challenges, where does he see acomplishment.
```
Comment (rufuspollock) [inline notes/democracy.earth.md:9]: Is this an intentional addition? Not sure it should be part of this commit.

File: notes/decentralization-autonomy-organisation-brekke.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+- "Decentralization as a strategy not ideology"
+- "like moxie said admiting that not everyone is going to run his own server/node"
+- "I just want to open up these concepts so that they are considered less as ideologies less as kind of absolute goods and more as as as strategies as as pragmatics to try and achieve something so that we can actually start to be a little bit more clear on the things that we're actually really trying to achieve"
+- "I like to come back and end up with the policy report that i wrote for for the european commission because it was really some a kind of process where i was thinking .. whether the arts might be a kind of gateway for people who are involved in crypto to actually start experimenting more with some actually functioning and upgraded forms of democracy for the digital age"  
+- euopean comission report: https://publications.jrc.ec.europa.eu/repository/handle/JRC121675 : d
+SCANNING THE EUROPEAN ECOSYSTEM OF DISTRIBUTED LEDGER FOR SOCIAL AND PUBLIC GOOD TECHNOLOGIES - What, Why, Where, How, and Ways to Move Forward - Alexandre Pólvora (editor), Anna Hakami (editor),Erica Bol (editor) ,Samer Hassan,
```
Comment (rufuspollock) [inline notes/decentralization-autonomy-organisation-brekke.md:55]: Should this and next few lines be one line?

**Review by rufuspollock — CHANGES_REQUESTED** [review]
Generally looks very good and would love to have this in. wdyt of doing a small bit of tidying on the raw notes to make them a bit more readable and tidy. i've also flagged some specific points too?

File: notes/decentralization-autonomy-organisation-brekke.md (hunk 4 above)
Comment (AndreasS2501) [reply notes/decentralization-autonomy-organisation-brekke.md]: Hi @rufuspollock I changed all the things you mentioned here on my main branch so it should be fine now 👍

File: notes/decentralization-autonomy-organisation-brekke.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
+# Keynote Decentralization, Autonomy and Organization - jaya klara brekke
+
```
Comment (rufuspollock) [inline notes/decentralization-autonomy-organisation-brekke.md:2]: Could we provide a bit of context about where this keynote took place ...

File: notes/decentralization-autonomy-organisation-brekke.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+markets and financialization"
+- ".. but we can't really afford that luxury anymore and we need to build at least as much as we critique at this time to really kind of set 
+the ground for generations in the future"
+- QA
+- JKB: resist the appeal of the magic diagram that "solves" governance ( a good hint at process dynamics), put emphazise on relationshsips 
+- ".. using decentralization in order to rebuild and reinstate some form of global solidarities"
+- practical example: decentralization and privacy (work at nim) 
+- Ruth: "we have way more questions than we could possibly answer maybe even in a day" so how to meaningfully decentralize the event of a keynote? How to create a more participatory and perspectival experience about the intent of the key note? Use facilitation to split groups to share expectations and experience before the keynote and then after wards too? Need experiments here too! How to embed (small)experiences of the topic of the keynote into a digital possibility 
+- Q:  Can crypto and capital be a means to achieve some post-capitalist?
```
Comment (rufuspollock) [inline notes/decentralization-autonomy-organisation-brekke.md:61]: Heading? e.g. third level

File: notes/decentralization-autonomy-organisation-brekke.md (hunk 5 above)
Comment (AndreasS2501) [reply notes/decentralization-autonomy-organisation-brekke.md:2]: sure it is in the third line:

=>  Radical Friends. #HausDerKunst #GoetheInstitut DAO Summit for Decentralisation of Power and Resources in the Artworld

Thats the name of the event

Comment (rufuspollock) [reply notes/decentralization-autonomy-organisation-brekke.md:2]: OK, i would suggest moving that up somewhere prominent e.g. the first line of the whole piece. Also add a date etc.

**Pull request merged**